playback/src/player.rs

```diff
@@ -251,33 +247,33 @@ impl Player {
         let (cmd_tx, cmd_rx) = mpsc::unbounded();
         let (event_sender, event_receiver) = mpsc::unbounded();
 
-        debug!("new Player[{}]", session.session_id());
-
-        let internal = PlayerInternal {
-            session: session,
-            config: config,
-            commands: cmd_rx,
-
-            state: PlayerState::Stopped,
-            preload: PlayerPreload::None,
-            sink: sink_builder(),
-            sink_status: SinkStatus::Closed,
-            sink_event_callback: None,
-            audio_filter: audio_filter,
-            event_senders: [event_sender].to_vec(),
-        };
+        let handle = thread::spawn(move || {
```

Comment:
I'm not too familiar with this code but why does this use `thread::spawn` as opposed to using a tokio task? Or rather, if it's using blocking code, should it not be spawned with `spawn_blocking` into the tokio blocking threadpool?

Reply:
This is definitely blocking (I tried to rewrite it as future, but encountered strange bugs). Also it's a long-running thread, powering the playback, so an own thread is ok, I think.